Lib/bdb.py

```diff
@@ -365,6 +367,12 @@ def set_quit(self):
     # Call self.get_*break*() to see the breakpoints or better
     # for bp in Breakpoint.bpbynumber: if bp: bp.bpprint().
 
+    def _add_to_breaks(self, filename, lineno):
+        """Add breakpoint to breaks, if not already there."""
+        list = self.breaks.setdefault(filename, [])
```

Comment:
We should use a more meaningful name here, and avoid shadowing the built-in `list`.

Reply:
oh! yeah